ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -342,12 +342,19 @@ calcParserModules ghcFlavor = do
       -- Lastly, manipulate text like 'GHC/Exts/Heap/Constants.hs'
       -- into 'GHC.Exts.Heap.Constants'.
       modules = map (replace "/" "." . dropSuffix ".hs") strippedModulePaths
-      -- The modules in this list would by default end up in
-      -- ghc-lib. We intervene so that rather, they go into
-      -- ghc-lib-parser.
+      -- The modules in this list elude being listed in
+      -- '.parser-depends' but are required by ghc-lib-parser. We
+      -- intervene and patch things up.
       extraModules =
-        [ "GHC.Driver.Config" | ghcFlavor > Ghc901 ] ++
-        [ "GHC.Parser.Errors.Ppr" | ghcFlavor > Ghc901 ] ++
+        [ x | ghcFlavor > Ghc901
```

Comment:
why do we need this for 9.2 as well? I would have expected only HEAD to change

Reply:
you are quite right. thanks. stand by, fix on its way

Reply:
should be sorted now @cocreature